parsely/tests.py

```diff
@@ -155,6 +157,37 @@ def test_invalid_ref_type(self):
             pass
         self.assertTrue(r == "sentinel")
 
+    def test_analytics_sort(self):
+        r = self.p.analytics(aspect="posts", sort="mobile_views")
+        self.assertTrue(r[0].mobile_views > 0)
+
+        def handle(res):
+            self.assertTrue(res[0].mobile_views >0)
+        self.p.analytics(aspect="posts", sort="mobile_views", _callback=handle)
+
+    def test_related_boost(self):
+        r = self.p.related(self.train_link, boost="visitors_returning")
+        self.assertTrue(r[0].title != "")
+
+        def handle(res):
+            self.assertTrue(res[0].title != "")
+
+        self.p.related(self.train_link, boost="visitors_returning", _callback=handle)
+
+    def test_search_boost(self):
+        s = self.p.search("security", limit=4, boost="visitors_new")
+        self.assertTrue(s[0].title != "")
+
+        def handle(res):
+            self.assertTrue(res[0].title != "")
+
+        self.p.search("security", limit=4, boost = "visitors_new", _callback=handle)
+
+    def test_wrong_metric(self):
+        try:
+            r = self.p.analytics(aspect="posts", sort="Mobile_view")
+        except (ValueError):
```

Comment:
unit tests actually has a function you can use here for this: 

https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertRaises

that way the test only passes if the exception you specify happens. In this formulation, if someone were to break the valid_kwarg decorator by accident, this test would still pass